sdk/keyvault/azure-security-keyvault-keys/src/key_client.cpp

```diff
@@ -23,6 +23,8 @@ using namespace Azure::Core::Http::Policies;
 using namespace Azure::Core::Http::Policies::_internal;
 
 namespace {
+constexpr static const char KeyVaultServicePackageName[] = "keyvault-keys";
```

Comment:
Do other languages avoid using the full name "azure-security-keyvault-keys" for telemetry? What is the exact `service name`?

And should it be all lower-case?

@antkmsft, should we lower-case the `i` here?
https://github.com/Azure/azure-sdk-for-cpp/blob/fda88bf7b8389575b9d973821efc604442e10493/sdk/identity/azure-identity/src/client_secret_credential.cpp#L97

Reply:
For reference, this is what storage does:

https://github.com/Azure/azure-sdk-for-cpp/blob/fda88bf7b8389575b9d973821efc604442e10493/sdk/storage/azure-storage-common/inc/azure/storage/common/internal/constants.hpp#L7-L10

Reply:
@heaths what is the telemetry string for KeyVault Keys in .NET?
https://github.com/Azure/azure-sdk-for-net/blob/4162f6fa2445b2127468b9cfd080f01c9da88eba/sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs#L106-L118